subworkflows/nf-core/vcf_gather_bcftools/main.nf

```diff
@@ -15,10 +15,10 @@ workflow VCF_GATHER_BCFTOOLS {
     ch_versions = Channel.empty()
 
     ch_concat_input = ch_vcfs.join(ch_scatter_output)
-        .map{ meta, vcf, tbi, bed, gather_count ->
+        .map{ meta, vcf, tbi, _bed, gather_count ->
             meta = val_common_meta ? meta + [id:meta[val_common_meta]] : meta
             [ groupKey(meta, gather_count), vcf, tbi ]
-        }.groupTuple()
+        }.groupTuple( sort: true )
```

Comment:
Not sure this helps.
In some contexts, you end up sorting on the full path, and because they the work directory, you end up sorting on the whole file path rather than just the name.

Reply:
I deleted it